src/pages/home/WeatherTable.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+
+class WeatherTable extends Component {
+    constructor() {
+        super();
+        this.state = {
+            isLoading: false,
+            eRRor: null,
+            input: '',
+            zipCode: 'N/A',
+            currentWeather: {
+                main: {
+                    temp: '255.38',
+                    humidity: '',
+                    temp_min: '255.38',
+                    temp_max: '255.38',
+                },
+                name: '',
+                wind: {
+                    speed: '',
+                },
+                weather: [{
+                    description: '',
+                }],
+            },
+        }
+    }
+
+    onInputChange=(event) => {this.setState({input: event.target.value})};
+
+    onButtonSubmit = () => {
+        this.setState({isLoading: true});
+        this.setState({zipCode: this.state.input});
+        fetch(`https://api.openweathermap.org/data/2.5/weather?zip=${this.state.input},us&APPID=7f838af83bf39591f00de86b6f7329e1`)
+            .then(response=> {
+                if (response.ok) {return response.json();}
+                else {throw new Error(`Invalid Zip Code:  ${this.state.input}.`);}})
+            .then(data => this.setState({ currentWeather: data,isLoading: false}))
+            .catch(eRRor => this.setState({eRRor,isLoading: false}));
+    };
+
+    refreshPage = () => {window.location.reload(true)};
+
+    render() {
+        const {currentWeather,zipCode,isLoading,eRRor} = this.state;
+
+        if (eRRor) {return (<div className="text-center">
+            <p>{eRRor.message}</p>
+            <button type="button" value="Refresh" className="btn btn-secondary" onClick={this.refreshPage}>Refresh</button>
+            </div>)};
+
+        if (isLoading) {return (<div className="text-center"><p>Loading...</p></div>)};
+
+        var windSpeed = (currentWeather.wind.speed)*2.237;
+        var highTemp = (currentWeather.main.temp_max-273.15)*9/5+32;
+        var currentTemp = (currentWeather.main.temp-273.15)*9/5+32;
+        var lowTemp = (currentWeather.main.temp_min-273.15)*9/5+32;
```

Comment:
You're repeating logic here three times. Should probably move this to a function and pass in the temp for a return value.